dashboard/src/lib/index.ts

```diff
@@ -47,16 +47,16 @@ export async function getDevices(creatorId: string): Promise<Device[]> {
     }
 }
 
-export async function createDevice(): Promise<void> {
+export async function createDevice(deviceName: string): Promise<void> {
     try {
         // Create a channel
         // Timeout to see progress bar animation (from 0 to 0.33)
         setTimeout(() => deviceCreationProgress.set(0.33), 100)
-        const channel = await createChannel([{ type: 'cxc', source: 'cxc' }]);
+        const channel = await createChannel(`channel-${deviceName}`,'',[{ type: 'cxc', source: 'cxc' }]);
```

Comment:
the logic of composition of channelName need to be in a separate method is used in more than one place.